content/guides/code-splitting-libraries.md

```diff
@@ -185,7 +185,7 @@ module.exports = function(env) {
 };
 ```
 
-With the above webpack config, we see three bundles being generated. `vendor`, `main` and `manifest` bundles. 
+With the above webpack config, we see three bundles being generated. `vendor`, `main` and `manifest` bundles.
```

Comment:
"With the above webpack config, we see three bundles being generated: `vendor`, `main` and `manifest`."
Might read a little nicer.